demo/auth/passport.js

```diff
@@ -0,0 +1,47 @@
+var $fh = require('fh-js-sdk');
+var logger = require('@raincatcher/logger').logger;
+var ConsoleLogger = require('@raincatcher/logger').ConsoleLogger;
+var setLogger = require('@raincatcher/logger').setLogger;
+
+$fh.init({}, function() {
+  var passport = {
+    loadUserProfile: function($http, $window) {
+      var req = {
+        method: 'GET',
+        url: $fh.getCloudURL() + '/profile'
+      };
+      return $http(req, {withCredentials: true}).then(function (res) {
+        return res.data;
```

Comment:
This is related to the offline epic tickets and can be done later separately. I would be against storing the user profile in localstorage as it provides no access control. We could just show a placeholder user icon and "Offline" text for the profile data UI element when offline.